<ui version="4.0" >
 <class>AutoOptForceField</class>
 <widget class="QWidget" name="AutoOptForceField" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>202</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AutoOpt Force Field</string>
  </property>
  <widget class="QWidget" name="horizontalLayout" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>332</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <item>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Force Field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ForceFieldComboBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text" >
          <string>Ghemical</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Number of steps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="StepsSpinBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>5</number>
        </property>
        <property name="value" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Algorithm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="AlgorithmComboBox" >
        <item>
         <property name="text" >
          <string>Steepest Descent</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Conjugate Gradients</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Gradients</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="GradientsComboBox" >
        <item>
         <property name="text" >
          <string>Numerical</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Analytical</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Convergence</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ConvergenceSpinBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="suffix" >
         <string/>
        </property>
        <property name="prefix" >
         <string>10e-</string>
        </property>
        <property name="minimum" >
         <number>2</number>
        </property>
        <property name="maximum" >
         <number>10</number>
        </property>
        <property name="value" >
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_buttonStartStop" >
        <property name="text" >
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
